### DISCUSSION TITLE
Lesson 11 NFTs, Can't find the Token ID

### DISCUSSION THREAD

**Discussion by daryle-tan** [Cyfrin/foundry-full-course-cu#262] [Q&A] [answered]
Hello. I am on lesson 11 at 8:34:00 of the second video. I am trying to import the PUG nft into my metamask but I don't think I have the correct Token ID. Here is a screenshot of what I am importing.
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/95397972/755712ae-f6c0-4647-8e35-14ed18236e4d)

**Comment by shikhar229169** [comment]
Ensure that you are adding the correct contract address associated with NFT, and also you have chosen the correct account in metamask from which you minted the NFT.

**Reply by shikhar229169** [reply]
Can you drop the contract address associated with the NFT, along with your account public address in which you are trying to add NFT in metamask.

**Reply by daryle-tan** [reply]
Thanks for the reply. Here's the contract address 0xC45bAB6130e6eAe24D8309BaFc753eEC3dD2493f
and my public address from metamask:
0x3ad65239A011766a4F6466252151B4a196Ef5CBF

**Reply by daryle-tan** [reply]
I think I might've used a different private key in my .env file to mint the contract

**Reply by Naman1729** [reply]
Use this Account "0xeE3de49D2FeeBe6B362606E19E18cb4D971115c0" to import you NFT .

**Reply by shikhar229169** [reply]
@daryle-tan Yes, the account should be same, from which you mint the NFT, and the account in which you want to add NFT.

**Reply by daryle-tan** [reply]
@Naman1729  thank you as well!

**Comment by daryle-tan** [comment]
> Use this Account "0xeE3de49D2FeeBe6B362606E19E18cb4D971115c0" to import you NFT .

That didn't work either. I tried updating the private key in my .env and redeployed it, and reminted. I used the new contract address and token ID of 0 but no luck.

**Comment by shikhar229169** [comment, marked as answer]
I can see that in txn you minted the NFT by address ```0x3ad65239A011766a4F6466252151B4a196Ef5CBF```, so try adding NFT in that. Pass the contract address ```0xa2ed168e2f55B5e2E9aeBeeA0077DB54C8f162C6```, and tokenID = 0.

**Reply by daryle-tan** [reply]
That was it. Thank you!
